firmware-management/swupdate-handlers/arm-handler-common.h.in

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+*/
+
+#include "swupdate/swupdate.h"
+
```

Comment:
Should have an `#include` to get `size_t`.

Reply:
Yes, it should... I had missed this one.